### PR TITLE
chore: added pull request template file

### PR DESCRIPTION
**Pull request by luke-mcfarlane-rocketlab**
## JIRA Ticket

[BSS-197](https://jira.csiro.au/browse/BSS-197)

## Description

Create a PR template for FAIMS3 repo.

## Proposed Changes

- Created PR template file

## How to Test

- Review pull_request_template.md file (happy to take any suggestions/edits)
- I don't think it is possible to test that it gets used automatically until it is merged into the main repository.

## Checklist

- [x] I have confirmed all commits have been signed.
- [x] I have added JSDoc style comments to any new functions or classes.

### CODE REVIEW

**Comment by PeterBaker0** [comment]
@luke-mcfarlane-rocketlab  all good for now but if you want to adhere to semver in the PR title this would be a chore probably not feat (as it shouldn't promote a new software version), and you have to use lower case for the whole first part, plus start with lower case for the detail. Have updated as such

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+## JIRA Ticket
```
Comment (PeterBaker0) [inline .github/pull_request_template.md:1]: Could you add a title at the top like 

\# feat(scope): pr title in semver format

**Comment by luke-mcfarlane-rocketlab** [comment]
> @luke-mcfarlane-rocketlab all good for now but if you want to adhere to semver in the PR title this would be a chore probably not feat (as it shouldn't promote a new software version), and you have to use lower case for the whole first part, plus start with lower case for the detail. Have updated as such

Awesome, thanks!

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+Explain how to test the changes.
+
+## Additional Information
+
+Add any additional information or context about the pull request here.
+
+## Checklist
+
+- [ ] I have confirmed all commits have been signed.
+- [ ] I have added JSDoc style comments to any new functions or classes.
```
Comment (PeterBaker0) [inline .github/pull_request_template.md:24]: Could we add something about documentation e.g. README(s) updated, developer process guides updated etc. Maybe

"Relevant documentation such as READMEs, guides, and class comments are updated?"

**Review by PeterBaker0 — APPROVED** [review]
Fine with me - just a few comments for minor improvements. 

Thanks.

File: .github/pull_request_template.md (hunk 1 above)
Comment (luke-mcfarlane-rocketlab) [reply .github/pull_request_template.md:3]: Actioned

File: .github/pull_request_template.md (hunk 2 above)
Comment (luke-mcfarlane-rocketlab) [reply .github/pull_request_template.md:26]: Actioned

**Review by stevecassidy — APPROVED** [review]
Looks good to me.

**Comment by stevecassidy** [comment]
On second thoughts, I think we need a good definition of 'semver format' so we can be sure to get it right.

**Comment by luke-mcfarlane-rocketlab** [comment]
> On second thoughts, I think we need a good definition of 'semver format' so we can be sure to get it right.

Updated - removed the mention. I agree, I think it is clearer now

**Pull request merged**